examples/app-api/README.md

```diff
@@ -0,0 +1,65 @@
+# App Router Headless API
```

Comment:
Should this be deleted as the example is removed besides this and `.gitignore` now?